src/models/data.cpp

```diff
@@ -133,6 +133,158 @@ ItemCost buildCallerCalleeResult(const BottomUp& data, const Costs& bottomUpCost
     }
     return totalCost;
 }
+
+bool startsWith(const QStringRef& str, const QLatin1String& prefix, int* size)
```

Comment:
static, or into anon namespace, or make it a lambda in the function where it's used